pkg/stanza/operator/input/tcp/tcp.go

```diff
@@ -102,8 +117,12 @@ func (c Config) Build(logger *zap.SugaredLogger) (operator.Operator, error) {
 		return nil, err
 	}
```

Comment:
do we still need these blocks now that its moved into `defaultMultilineBuilder`

Reply:
yes that encoding is for the tcp object.

Reply:
can we reuse the encoding object instead of rebuilding it in `defaultMultilineBuilder` then?
```
func (c Config) defaultMultilineBuilder(encoding helper.Encoding) (bufio.SplitFunc, error) {
```